digitaltwin/service/readme.md

```diff
@@ -0,0 +1,42 @@
+# Azure IoT Digital Twins Service SDK
```

Comment:
Is this readme.md different than the earlier one? It looks more or less the same, at first glance.